### ISSUE TITLE
Interest for Okabe-Ito color palette?

### ISSUE THREAD

**Issue by vigou3** [latex3/xcolor#35]
Inspired by its availability in R graphics, I'm increasingly using the [Okabe-Ito universal color palette](https://jfly.uni-koeln.de/color/#pallet) in my documents. Would there be any interest to add the seven colors to the **xcolors**? I can contribute the code. Otherwise, I might just make a tiny separate package.

**Comment by josephwright** [comment]
You don't need a package to set up some colours: the existing ones in `xcolor` are essentially historical (beyond the base names).

**Comment by vigou3** [comment]
Yeah, I know, but just loading a package to have colours available instead of setting them up in the preamble remains convenient, isn't it? Hence the proposal.

**Comment by simon-pfahler** [comment]
Defining "convenient" colors is not the job of `xcolor`, but there are packages that do that.
I have written the package `colorblind`, which does what you want (and more): [https://github.com/simon-pfahler/colorblind](https://github.com/simon-pfahler/colorblind)
It is also available in CTAN.

Your would only need to add the following line to the preamble:
```
\usepackage[OkabeIto]{colorblind}
```

The colors are then available via the names `OI0`, `OI1`, ..., `OI7`.

**Comment by vigou3** [comment]
Ah, great Simon! I'll then use `colorblind` henceforth. Case closed. Thanks!
